samples/ChangeFeedMigrationTool/DocumentFeedObserverFactory.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Azure.Documents.ChangeFeedProcessor;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+
+public class DocumentFeedObserverFactory: IChangeFeedObserverFactory
+{
+    DocumentClient client;
+    DocumentCollectionInfo collectionInfo; 
+
+    public DocumentFeedObserverFactory(DocumentCollectionInfo destCollInfo)
```

Comment:
It's OK like this but I would recommend to pass DocumentClient instance to factory. This way factory is not responsible for logic of creation of the client and checking whether it needs to see if database/collection exists, etc.